assets/src/stories-editor/blocks/amp-story-cta/deprecated.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { getClassNameFromBlockAttributes, getStylesFromBlockAttributes } from '../../helpers';
+
+const blockAttributes = {
+	url: {
+		type: 'string',
+		source: 'attribute',
+		selector: 'a',
+		attribute: 'href',
+	},
+	text: {
+		type: 'string',
+		source: 'html',
+		selector: 'a',
+	},
+	customTextColor: {
+		type: 'string',
+		default: '#ffffff',
+	},
+	customBackgroundColor: {
+		type: 'string',
+		default: '#32373c',
+	},
+};
+
+const deprecated = [
+	{
+		attributes: {
+			align: {
+				type: 'string',
+				default: 'center',
+			},
+			...blockAttributes,
+		},
+		supports: {
+			align: true,
+			alignWide: false,
+		},
+		save( { attributes } ) {
+			const {
+				url,
+				text,
+			} = attributes;
```

Comment:
@swissspidy Any thoughts on how to add `propTypes` to the `attributes` here within this array of objects without creating a separate `const` for `save`?

Reply:
I don't think that's possible. No harm in having a separate `const` for it.

Reply:
Added within f1f0cdf.